core/src/setup.rs

```diff
@@ -41,7 +41,7 @@ pub async fn node_setup(
 
     // TODO: remove this once we have a more fleshed
     // out startup flow
-    tokio::time::sleep(Duration::from_secs(15)).await;
+    tokio::time::sleep(Duration::from_secs(5)).await;
```

Comment:
going to remove this pretty soon anyways, we don't need to wait for a leader election, this is actually more important for the followers now.